lerna.json

```diff
@@ -1,7 +1,7 @@
 {
-  "lerna": "2.0.0-rc.5",
+  "lerna": "2.0.0",
   "packages": [
     "packages/*"
   ],
-  "version": "1.3.0-rc2"
+  "version": "independent"
```

Comment:
Why do we need independent versionning here? This will make handling of version compatibility more challenging because of the coordination needed between different version numbers IMO.

Reply:
I understand your point, on the other hand, I think this is confusing that a package changes version because another one has some modification where the first one has not changed at all.

Reply:
if I remember correctly it only changes if the package has modifications, but bumps then are incoherent